src/tests/ftest/daos_test/daos_core_test_dfs.yaml

```diff
@@ -0,0 +1,59 @@
+# change host names to your reserved nodes, the
+# required quantity is indicated by the placeholders
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+    - server-C
+    - server-D
+  test_clients:
```

Comment:
is this the max number of client nodes you can use (i.e. the more the better)?

Reply:
not at the expense of the number of servers. was just thinking, like is 6 total the max?

Reply:
On Large cluster we have 8 total nodes so I can make total clients as 4. But you mentioned above you are fine to run this with single client which is a bit confusing. So you want me to have it run with single client multiple processes or multiple clients multiple processes?
Also note, currently we are running daos_core_test on medium cluster which has a max nodes of 4 and we are using them all for servers.
As this particular use case need to be run with more clients I moved only this test to large cluster.
If it's fine to have single client multiple processes, I can keep this within daos_core_test.py and just make the number of processes to 16.
But if it needs to be multi client multi processes, we need large hw cluster and I can increase the clients to 4. You may let me know if you want me to increase the num of processes from 16 to any other number in that case though.

Reply:
no sorry that was not my intention. we do want to run this on multiple client nodes. I was asking here if we can do more than 2.
For my comment above, i didn't mean as a replacement. i just meant it's fine to let this test run twice, once with a single rank like it is today (don't need that change aboce) and once with this mode in parallel. but it's probably just redundant.

Reply:
yeah we can do that. I will leave it in daos_core_test as is. And will add 2 more client nodes for daos_core_test_dfs, which will make total clients as 4. Any suggestion for total number of client processes?

Reply:
let's do 32 to exercise the conditionals more

Reply:
Done